app/services/agreements/reports/partner_api_report.rb

```diff
@@ -3,15 +3,6 @@ module Reports
     class PartnerApiReport < ApplicationJob
       queue_as :low
 
-      include GoodJob::ActiveJobExtensions::Concurrency
-
-      good_job_control_concurrency_with(
```

Comment:
I think we have specs for most of these concurrency key methods, so we probably need to delete those too